src/ray/core_worker/core_worker.cc

```diff
@@ -131,6 +132,9 @@ CoreWorker::CoreWorker(const WorkerType worker_type, const Language language,
       raylet_socket, WorkerID::FromBinary(worker_context_.GetWorkerID().Binary()),
       (worker_type_ == ray::WorkerType::WORKER), worker_context_.GetCurrentJobID(),
       language_, worker_server_.GetPort()));
+  if (direct_actor_task_receiver_ != nullptr) {
+    direct_actor_task_receiver_->Init(*raylet_client_);
```

Comment:
Why not just pass this into the constructor?

Reply:
Figured this out.

Reply:
Oh actually, we still need this. The reason is the raylet client cannot be constructed prior to the receivers.

Reply:
Why can't raylet client be constructed before the receivers? Leave a comment pls.